packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -64,9 +65,30 @@ const entryPointExists = (segments) => {
     return false
 }
 
-const findInProjectThenSDK = (pkg) => {
+/**
+ * Find the path to a dependency
+ * First check the project's node_modules, then the SDK's node_modules, then the monorepo's node_modules
+ * @param {string} pkg - name of the dependency
+ * @returns {string} path to the dependency
+ */
+const findDependency = (pkg) => {
     const projectPath = resolve(projectDir, 'node_modules', pkg)
-    return fs.existsSync(projectPath) ? projectPath : resolve(sdkDir, 'node_modules', pkg)
+    if (fs.existsSync(projectPath)) {
+        return projectPath
+    }
+    const sdkPath = resolve(sdkDir, 'node_modules', pkg)
+    if (fs.existsSync(sdkPath)) {
+        return sdkPath
+    }
+    if (isMonoRepo) {
```

Comment:
As you mentioned in your PR description, I don't think we have considered the scenario of developers having their own monorepo. So I'd consider this to be a new feature.

However, there is a bug with the current implementation: for those developers who do not have a monorepo, the `sdkDir` is actually _not_ resolved correctly.

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/24b8b16407ac9f3ab4743c9294b899ff7fbc7eac/packages/pwa-kit-dev/src/configs/webpack/config.js#L26

The above line would resolve correctly only for those folks developing PWA Kit:
- Within PWA Kit monorepo: `packages/pwa-kit-dev/src/configs/webpack/config.js` → `sdkDir` is mapped to `packages/pwa-kit-dev/`
- Within a generated project (and no monorepo): `node_modules/pwa-kit-dev/configs/webpack/config.js` → `sdkDir` is mapped to `node_modules`